option optimize_for = LITE_RUNTIME;

message BuyMonthCardRequest
{
 required uint32 id = 1;
}
